### ISSUE TITLE
Wrong INSERT statement detection in ADODB_mssqlnative::_query method

### ISSUE THREAD

**Issue by hondzik** [ADOdb/ADOdb#155]
Hi,
INSERT statement detection in ADODB_mssqlnative::_query is not working if there is a new line and/or tab character before "INSERT" word in $sql string:
```php
$sql = '
	INSERT INTO table (col1, col2, col3)
	VALUES (val1, val2, val3)
';
```
Because of that, "select scope_identity()" is not added to the $sql string so you can`t retrieve Insert_ID.
I am using ADOdb 5.19 and Windows (CRLF) new line characters.

**Comment by dregad** [comment]
As far as I can tell, [the regex in mssqlnative `_query()` method](https://github.com/ADOdb/ADOdb/blob/v5.19/drivers/adodb-mssqlnative.inc.php#L566) `/^\W*(insert [^;]+);?$/i` returns 1 given the above-mentioned SQL query, regardless of line endings (CRLF or LF).

Please note that this regex was modified in 067b14d9d7e68c7dc49a1a874dd566e962eb1335 (not yet released, will be part of upcoming 5.20), but that also returns 1.

Short version: I can't reproduce the problem.

**Comment by mnewnham** [comment]
I confirm, I can't reproduce this either. @hondzik , could you perhaps provide a 'real' code sample for us to review?

**Comment by dregad** [comment]
You have not provided any feedback, I assume this is no longer an issue and will therefore close this. Feel free to reopen if necessary.
